pkg/cmd/pulumi/deployment_settings_config.go

```diff
@@ -645,12 +687,14 @@ func configureGitSSH(ctx context.Context, be backend.Backend, s backend.Stack,
 
 	var password string
 
-	if password, err = cmdutil.ReadConsoleNoEcho("Enter the private key password []"); err != nil {
+	password, err = promptForValue(d.Yes, "(optional) private key password", password, true,
```

Comment:
If this is a prompt, I think we should start with a capital letter.

Reply:
are you sure? Cant tell if all the prompts follows this pattern but I have seen many being all lowercase https://github.com/pulumi/pulumi/blob/a1d6b2cc162a46dcc44b2ba4ce159873cb4c6ce0/pkg/cmd/pulumi/up.go#L276-L294

Reply:
I capitalized the prompts after discussing it!